frontend/src/metabase/components/NewItemMenu/NewItemMenu.tsx

```diff
@@ -136,9 +138,10 @@ const NewItemMenu = ({
         title: t`Metric`,
         icon: "metric",
         link: Urls.newQuestion({
-          mode: "notebook",
-          collectionId,
+          mode: "query",
+          creationType: "custom_question",
```

Comment:
Let's remove this until it's needed for analytics.

Reply:
d07a15c1e79b1362757af80cb222064938014aa7